cid/builtin/core/data/queries/shared/account_map.sql

```diff
@@ -1,10 +1,12 @@
 CREATE OR REPLACE VIEW account_map AS
 SELECT 
     ${account_id} AS account_id,
-    MAX_BY(${account_name}, concat(${account_name}, ': ', ${account_id}) AS account_name
-FROM 
+    MAX_BY(${account_name}, concat(${account_name}, ': ', ${account_id})) AS account_name,
```

Comment:
Here is fixed missing closing bracket. Without fix, dashboard deployment will crash.